### PR TITLE
Add landing page demo

### PR DESCRIPTION
**Pull request by toddbaert**

### CODE REVIEW

File: packages/app/src/main.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { AppModule } from './app/app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  app.enableCors();
```
Comment (beeme1mr) [inline packages/app/src/main.ts:13]: Do you need this?

File: flags.json.example [hunk 2]
```diff
@@ -2,6 +2,17 @@
   "newWelcomeMessage": {
     "state": "enabled"
```
Comment (beeme1mr) [inline flags.json.example:3]: This should start out as disabled.

**Review by beeme1mr — COMMENTED** [review]
This is a great start but I ran into one issue. When you click off the editor to apply the change, it closes the tour.

File: packages/app/src/main.ts (hunk 1 above)
Comment (toddbaert) [reply packages/app/src/main.ts:13]: Yes. The UI and backend run on different ports, the browser considers that cross-origin.

File: flags.json.example (hunk 2 above)
Comment (toddbaert) [reply flags.json.example:3]: agreed.

Comment (toddbaert) [reply flags.json.example:3]: fixed.

**Comment by toddbaert** [comment]
> The UI also bombs out if the server isn't running.

Added a more helpful error. I think we still want to fail loudly when this happens, but now users will see: `Unable to load page data... Did you forget to run the server?`

**Comment by toddbaert** [comment]
> This is a great start but I ran into one issue. When you click off the editor to apply the change, it closes the tour.

Fix this, the tour is ONLY closed  now when you click the 'x'.

**Comment by toddbaert** [comment]
Also added auto-advancing at both edit steps.

**Review by beeme1mr — APPROVED** [review]
Looks great!

**Pull request merged**